src/contracts/webauthn_account_contract/src/public_key_note.nr

```diff
@@ -13,19 +13,19 @@ use dep::aztec::{
     context::PrivateContext,
 };
 
-global PUBLIC_KEY_NOTE_LEN: Field = 3;
+global PUBLIC_KEY_NOTE_LEN: Field = 5;
 
 // Stores a public key composed of two fields
 // TODO: Do we need to include a nonce, in case we want to read/nullify/recreate with the same pubkey value?
 struct PublicKeyNote {
-    x: Field,
-    y: Field,
+    x: [u8; 32],
+    y: [u8; 32],
```

Comment:
Some considerations for the future:
1) Can we make it a type?
2) Would it be more efficient if do [Field; 2], where x = [x1, x2], x1 = x / p, x2 = x % p?
Feels like a lot of constraints.